src/ontology/cl-edit.owl

```diff
@@ -9205,13 +9205,15 @@ AnnotationAssertion(rdfs:label obo:CL_0000692 "terminal Schwann cell"^^xsd:strin
 SubClassOf(obo:CL_0000692 obo:CL_0000630)
 SubClassOf(obo:CL_0000692 obo:CL_0002376)
 
-# Class: obo:CL_0000693 (neuroglioform cell)
+# Class: obo:CL_0000693 (neurogliaform cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb"^^xsd:string) obo:IAO_0000115 obo:CL_0000693 "A stellate interneuron having 7-10 dendrites that may branch."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:17122314") obo:IAO_0000115 obo:CL_0000693 "An interneuron that has spider-like appearance with a small round soma, a large number (7-10) of short, smooth, or slightly beaded primary dendrites that give rise to only a few secondary branches, and a branched axon that establishes a dense axonal mesh with thin shafts."^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18568015") oboInOwl:hasExactSynonym obo:CL_0000693 "NGF cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:3235828") oboInOwl:hasExactSynonym obo:CL_0000693 "neuroglioform cell")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000693 "spiderweb cell"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:CL_0000693 "cell"^^xsd:string)
-AnnotationAssertion(rdfs:label obo:CL_0000693 "neuroglioform cell"^^xsd:string)
-SubClassOf(obo:CL_0000693 obo:CL_0000691)
+AnnotationAssertion(rdfs:label obo:CL_0000693 "neurogliaform cell"^^xsd:string)
+EquivalentClasses(obo:CL_0000693 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070001)))
```

Comment:
I removed it from asserted under "Stellate neuron" as
1) when looking at papers, they seem to be referring to different types of cell altogether
2) even if it should be under stellate, we can use PATO to create that hierarchy
@tgbugs would like input on this